File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/security/roles/dao/UserRolesRepository.java
package com.github.bumblebee.security.roles.dao;

import com.github.bumblebee.security.roles.domain.UserRoleRecord;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface UserRolesRepository extends CrudRepository<UserRoleRecord, Long> {

    List<UserRoleRecord> findByUserId(Long userId);

    int deleteByUserId(Long userId);
}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/security/roles/dao/UserRolesRepository.kt
package com.github.bumblebee.security.roles.dao

import com.github.bumblebee.security.roles.domain.UserRoleRecord
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface UserRolesRepository : CrudRepository<UserRoleRecord, Long> {

    fun findByUserId(userId: Long): List<UserRoleRecord>

    fun deleteByUserId(userId: Long): Int
}
